src/Components/JoinForm/JoinForm.js

```diff
@@ -66,4 +70,8 @@ const mapStateToProps = state => ({
   codes: state.codes,
 })
 
-export default connect(mapStateToProps, null)(JoinForm);
+const mapDispatchToProps = dispatch => ({
+  addNewCode: code => dispatch(addNewCode(code))
+})
+
+export default connect(mapStateToProps, mapDispatchToProps)(JoinForm);
```

Comment:
what is this line doing?

Reply:
This syntax and line connects the component to the redux store defined in the provider in src index.js